integration_tests/tests/assess/view_submitted_application.cy.ts

```diff
@@ -0,0 +1,53 @@
+//  Feature: Assessor views a submitted application
+//    So that I can assess a submitted application
+//    As an external NACRO assessor
+//    I want to view a particular submitted application
+//
+//  Scenario: follows an external link to the submitted application
+//    Given a submitted application exists
+//    And I am logged in as a NACRO assessor
+//    When I navigate to the url of the submitted application
+//    Then I see the assessor's read-only view of the submitted application
+//
+//  TODO
+//  Scenario: is redirected to the submitted application after logging in
```

Comment:
would this scenario be our responsibility to test, as isn't this functionality handled by HMPPS Audit middleware? They just take whatever the redirect param is and take the user there...

Reply:
good point!

Reply:
I've removed that scenario